tasks/bridge/deposits.ts

```diff
@@ -16,66 +16,90 @@ task(TASK_BRIDGE_DEPOSITS, 'List deposits initiated on L1GraphTokenGateway')
 
-    const startBlock = taskArgs.startBlock ? parseInt(taskArgs.startBlock) : 0
-    const endBlock = taskArgs.endBlock ? parseInt(taskArgs.endBlock) : 'latest'
-    console.log(`Searching blocks from block ${startBlock} to block ${endBlock}`)
+    console.log(
+      `Tracking 'DepositInitiated' events on L1GraphTokenGateway (${l1Gateway.address}) from block ${l1StartBlock} to block ${l1EndBlock}`,
+    )
+    console.log(
+      `Tracking 'DepositFinalized' events on L2GraphTokenGateway (${l2Gateway.address}) from block ${l2StartBlock} onwards`,
+    )
 
-    const events = await Promise.all(
+    const depositInitiatedEvents = await Promise.all(
       (
-        await gateway.queryFilter(gateway.filters.DepositInitiated(), startBlock, endBlock)
-      ).map(async (e) => ({
-        blockNumber: `${e.blockNumber} (${new Date(
-          (await graph.l1.provider.getBlock(e.blockNumber)).timestamp * 1000,
-        ).toLocaleString()})`,
-        tx: `${e.transactionHash} ${e.args.from} -> ${e.args.to}`,
-        amount: ethers.utils.formatEther(e.args.amount),
-        status: emojifyRetryableStatus(
-          await getL1ToL2MessageStatus(e.transactionHash, graph.l1.provider, graph.l2.provider),
-        ),
-      })),
+        await l1Gateway.queryFilter(l1Gateway.filters.DepositInitiated(), l1StartBlock, l1EndBlock)
+      ).map(async (e) => {
+        const retryableTicket = await getL1ToL2MessageReader(
+          e.transactionHash,
+          graph.l1.provider,
+          graph.l2.provider,
+        )
+
+        return {
+          l1Tx: `Block ${e.blockNumber} (${new Date(
+            (await graph.l1.provider.getBlock(e.blockNumber)).timestamp * 1000,
+          ).toLocaleString()}) ${e.transactionHash}`,
+          l2Tx: `${retryableTicket.retryableCreationId}`, // Can't get block data because of arb node rate limit
+          amount: prettyBigNumber(e.args.amount),
+          status: emojifyRetryableStatus(
+            await getL1ToL2MessageStatus(e.transactionHash, graph.l1.provider, graph.l2.provider),
+          ),
+        }
+      }),
     )
 
-    const total = events.reduce(
+    const total = depositInitiatedEvents.reduce(
       (acc, e) => acc.add(ethers.utils.parseEther(e.amount)),
       ethers.BigNumber.from(0),
     )
     console.log(
-      `Found ${events.length} deposits with a total of ${ethers.utils.formatEther(total)} GRT`,
+      `Found ${depositInitiatedEvents.length} deposits with a total of ${prettyBigNumber(
+        total,
+      )} GRT`,
     )
 
     console.log(
-      'L1 to L2 message status reference: 🚧 = not yet created, ❌ = creation failed, ⚠️  = funds deposited on L2, ✅ = redeemed, ⌛ = expired',
+      '\nL1 to L2 message status reference: 🚧 = not yet created, ❌ = creation failed, ⚠️  = funds deposited on L2, ✅ = redeemed, ⌛ = expired',
     )
 
-    printEvents(events)
+    printEvents(depositInitiatedEvents)
+    console.timeEnd('runtime')
+    console.timeLog('runtime')
```

Comment:
nit: TIL about `console.time` - but I tried locally and calling timeLog after timeEnd doesn't seem to work?

```
Welcome to Node.js v16.18.0.
Type ".help" for more information.
> console.time('foo')
undefined
> console.timeEnd('foo')
foo: 5.500s
undefined
> console.timeLog('foo')
undefined
> (node:4083) Warning: No such label 'foo' for console.timeLog()
(Use `node --trace-warnings ...` to show where the warning was created)

> 
```

Reply:
Ah good catch. You can use `timeLog` to log partial times before ending the timer with `timeEnd`. Don't remember why this one was here, probably a mistake.